server/startup.js

```diff
@@ -1,92 +1,147 @@
+            getUserName: function(id) {
+                var user = Meteor.users.findOne({
                     _id: id
+                });
+                if (user) {
+                    if (user.username) {
+                        return user.username;
+                    } else if (user.profile && user.profile.name) {
+                        return user.profile.name;
+                    } else if (user.emails && user.emails[0]) {
+                        return user.emails[0].address;
+                    } else {
+                        return "Signed In";
+                    }
+                }
+            },
+            clearImages: function() {
+                return Images.remove({});
+            },
+            clearGames: function() {
+                return Games.remove({});
+            },
+            //Creates new game with the given user ID as the creator
+            //TODO - Take config param
+            createGame: function(userId, faceUrl) {
+                var gameId = Games.insert({
+                    creatorId: userId,
+                    userIds: [userId],
+                    state: 1,
+                    config: {},
+                    name: ''
+                });
+
+                Rounds.insert({
+                    gameId: gameId,
+                    roundNumber: 1,
+                    imageToVotesMap: {},
+                    currentImage: faceUrl,
+                    state: "upload",
+                    result: []
+                });
+
+                return gameId;
+            },
+            //Adds the client's user to the game
+            joinGame: function(gameId) {
+                var currentGame = Games.findOne({
+                    _id: gameId,
+                    userIds: {
+                        $nin: [this.userId]
+                    }
+                });
+                if (currentGame) {
+                    currentGame.userIds.push(this.userId);
+                    Games.update({
+                        _id: gameId
+                    }, currentGame);
+                }
+            },
+            //Takes game _id and url for user's to match
+            //Creates new game round, and updates game to that round
+            newSession: function(gameId, faceUrl) {
+                var currentGame = Games.findOne({
+                    _id: gameId
+                });
+                Rounds.insert({
+                    gameId: gameId,
+                    roundNumber: currentGame.state + 1,
+                    imageToVotesMap: {},
+                    currentImage: faceUrl,
+                    state: "upload",
+                    result: []
+                });
+                currentGame.state++;
+                Games.update({
+                    _id: gameId
                 }, currentGame);
+            },
+            //Takes URL and round ID
+            //Creates new image and adds image to round
+            addImage: function(roundId, imageUrl) {
```

Comment:
I feel like some of this stuff could eventually end up in something like "game.js" or "round.js" know what I mean?